deploy-deps.sh

```diff
@@ -124,6 +132,11 @@ deploy_registry() {
     kubectl apply -k "${script_path}/dependencies/registry"
     retry "kubectl wait --for=condition=Ready --timeout=240s -n kind-registry -l run=registry pod" \
           "The local registry did not become available within the allocated time"
+    # Copy the registry secret to cert-manager ns so the ca cert can be distrubuted
+    kubectl delete secret -n cert-manager --ignore-not-found=true local-registry-tls
+    kubectl get secret local-registry-tls --namespace=kind-registry -o yaml \
+        | grep -v '^\s*namespace:\s' | kubectl apply --namespace=cert-manager -f -
```

Comment:
Does it have to be created in cert-manager namespace for the distribution?

Reply:
When deploying trust-manager, we specify a trusted namespace (only one). We can make it a different one. I was thinking whether to make it the namespace of the registry in the first place (i.e. deploy trust-manager for that specific namespace), but decided to go with what we have here on a call with Gal earlier today.

Reply:
Maybe we can create the certificate in `cert-manager` in the first place and it will be distributed to kind-registry as well?
This way we will not have to create a copy of it.
Do you think it might work?

Reply:
I was trying that, but it's not possible.
On the registry namespace, we need the private key for that certificate, and trust manager will not distribute private keys.
We can use trust manager to move the CA certificate to where we need it, but we cannot use it to move the tls.key  file to where the registry is created.

Reply:
Thanks for the explanation
LGTM